internal/pkg/aws/ec2/ec2.go

```diff
@@ -120,6 +121,25 @@ func ExtractVPC(label string) (*VPC, error) {
 	}, nil
 }
 
+// PublicIP returns the public ip associated with the network interface.
+func (c *EC2) PublicIP(ENI string) (string, error) {
```

Comment:
we do allcaps if it's an initialism that is exported  (https://github.com/golang/go/wiki/CodeReviewComments#initialisms)
```suggestion
func (c *EC2) PublicIP(eni string) (string, error) {
```

Reply:
you know i just realized and I am sorry about this, but our default security groups don't allow customers to connect to their tasks from the internet. So even if we give them their public IP addresses it's not much use 🤔  
It becomes useful with the `--security-groups` flag I think.

Anyway, we can still continue with this, it's just something to keep in mind in case a customer reaches out to us!

Reply:
^I am aware! They will need a security group with inbound rule that allow their connection. I think it may still be potentially useful for people that uses their own `--security-group`. Or we can add a flag such as`--show-public-ip` so that only people that need it see the output.